src/Tests/Microsoft.DotNet.ApiCompatibility.Tests/Microsoft.DotNet.ApiCompatibility.Tests.csproj

```diff
@@ -1,15 +1,16 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <ProduceReferenceAssembly>false</ProduceReferenceAssembly>
     <TargetFrameworks>net472;$(ToolsetTargetFramework)</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">$(ToolsetTargetFramework)</TargetFrameworks>
     <OutputType Condition="'$(TargetFramework)' == '$(ToolsetTargetFramework)'">Exe</OutputType>
-
-    <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
```

Comment:
Where is this set now?

Reply:
This was required when all projects were built as part of the repo's main sln file. These days, source build uses a different solution file for source build and doesn't attempt to build these projects anymore. The other test projects don't have that property anymore so I cleaned this up here.

Reply:
Sounds good, might be worth just checking with @MichaelSimons and other source build folks just to ensure this won't break them if we aren't sure.